### ISSUE TITLE
Missing `logstash_group` in the gemspec metadata

### ISSUE THREAD

**Issue by ph** [logstash-plugins/logstash-input-kafka#184]
There is a small typo in the plugin [metadata](https://github.com/logstash-plugins/logstash-input-kafka/blob/master/logstash-input-kafka.gemspec#L19):

We see a `group` key.
```ruby
  s.metadata = { 'logstash_plugin' => 'true', 'group' => 'input'}
```

But instead it should be a `logstash_group` key like this:
```ruby
  s.metadata = { 'logstash_plugin' => 'true', 'logstash_group' => 'input'}
```

This metadata information is used by the `list` subcommand to filter plugins based on their type, since we don't have the `logstash_group` key the Kafka input doesn't show up when we use `--group input` option.

**Comment by ph** [comment]
originally reported by @bosinm

**Comment by original-brownbear** [comment]
fixed in #207
